gtda/diagrams/features.py

```diff
@@ -590,10 +596,13 @@ class PersistentImage(BaseEstimator, TransformerMixin):
     _hyperparameters = {'sigma': [numbers.Number, (1e-16, np.inf)],
                         'n_values': [int, (1, np.inf)]}
 
-    def __init__(self, sigma, n_values=100, n_jobs=None):
+    def __init__(self, sigma, n_values=100,
+                 weight_function=lambda x: x[1],
```

Comment:
The default weight function should give more weight to points of high persistence than to points on low persistence.
What is the standard in the litterature?

Reply:
The function is applied to the *differentiated* multi-set {(b,d-b,q) \mid (b,d,q) \in PD}. Hence, this is what you mention: a weight proportional to the persistence.
The originally-proposed weighting function was a normalized version of this.  Normalize by the max birth value in the persistence diagram ( last equation on page 8 in [1507.06217](https://arxiv.org/pdf/1507.06217.pdf) ).

I can change this, but it requires a change in the API. That is, providing a function that takes a PD, and outputs a vector of (the correct shape) (nb_points,) or (nb_points,1). What do you think, in terms of ease-of-use?